java/client/src/org/openqa/selenium/remote/tracing/Span.java

```diff
@@ -25,6 +27,9 @@
   Span setAttribute(String key, Number value);
   Span setAttribute(String key, String value);
 
+  Span addEvent(String name);
+  Span addEvent(String name, Map<String,EventAttributeValue> attributeMap);
```

Comment:
Maybe I am missing context, but do we need to add `EventAttributeValue` and the wrappers? I see that [AttributeValue](https://javadoc.io/static/io.opentelemetry/opentelemetry-api/0.5.0/io/opentelemetry/common/AttributeValue.html) extends `Object` and adds some methods around it.
Could we have an approach where we do not need this extra classes and wrappers?

Reply:
I understand your thought here. I did consider using a Map<String,Object> but I thought it was not the best practise, because only String, Boolean,Number are allowed as attributes and wouldn't want the API caller to pass Objects that are not those. So thought of having wrappers  in addition to the wrappers helping with OpenTelemetry compatibility issues. What do you think would be a better way @diemol ?